crates/driver/src/infra/config/file/mod.rs

```diff
@@ -684,6 +688,11 @@ fn default_simulation_bad_token_max_age() -> Duration {
     Duration::from_secs(600)
 }
 
+// SKY lending DAI token
```

Comment:
Interesting. I wasn't aware that the contract @fedgiac used in the e2e test for the contract only worked with Dai. I'm fine to make this the default for the start where it's only relevant for the e2e test but we should have at least 1 supported general purpose flashloan lender.
Also this could indicate that we might need a module or component that suggests a good lending provider based on the token that needs to be borrowed. Should happen later, though (if at all).

Reply:
Agree. There are A LOT of assumptions made at the moment, but I have them in my list. Before going live, these will have to be addresses and some kind of validation for flashloan hints will be implemented to avoid weird cases.